site/content/docs/cli/constructs-reference.md

```diff
@@ -382,35 +382,34 @@ TcpAssertionBuilder.responseData().contains('ping')
 "{ source: 'RESPONSE_DATA', regex: '', property: '', comparison: 'CONTAINS', target: 'ping' }"
 ```
 
-## `CheckGroup`
+## `CheckGroupV2`
 
-You can explicitly organize Checks in Check Groups.
-
-This brings the following benefits:
+Use the CheckGroupV2 construct to organize your Checks into groups. This comes with the following benefits:
 
 1. Your Checks are organized in a folder in the Checkly web UI.
 2. You can trigger all Checks in a group from the web UI and via a command line trigger.
 3. You can manage group-level configuration like the runtime, activated & muted-state, tags and alert channels that trickle down to all the Checks in the group.
 
 > [!WARNING]
-> Adding a check to a group means having it _only_ alert through the group's alert channels. Make sure your group has connected alert channels, or you might miss out on important alerts!
+> Adding a check to a group that has defined group-level alert settings means the check will _only_ alert through the group's alert channels. Make sure your group has connected alert channels, or you might miss out on important alerts!
```

Comment:
```suggestion
> If you add a check to a group with group-level alert settings ON, the check will only send alerts through the group’s alert channels. Make sure the group has at least one alert channel connected, otherwise you won’t receive any alerts for that check.
```